pkg/epp/server/runserver.go

```diff
@@ -91,17 +88,8 @@ const (
 // NewDefaultExtProcServerRunner creates a runner with default values.
 // Note: Dependencies like Datastore, Scheduler, SD need to be set separately.
 func NewDefaultExtProcServerRunner() *ExtProcServerRunner {
-	poolGKNN := common.GKNN{
```

Comment:
I would keep everything as before, the only thing we need is a flag to indicate whether or not to start the reconcilers, encoding this as a groupkind doesn't seem right.